app/view/fund_transfer.py

```diff
@@ -0,0 +1,152 @@
+                                                bank_account_id=from_account.id,
+                                                transaction_type_id=transaction_type.id,
+                                                fund_id=output.get('id'),
+                                                transaction_status=transaction_status)
+            db.session.add(account)
+            db.session.commit()
+            account_transaction_detail_schema.dump(account)
+            logger.info("fund transfer data successfully created")
+            response = ResponseGenerator(data=output, message="fund transfer data successfully created",
+                                         success=True, status=status.HTTP_201_CREATED)
+            return response.success_response()
+        except Exception as error:
+            logger.exception(error)
+            response = ResponseGenerator(data={}, message="Missing or sending incorrect data to create an activity",
+                                         success=False, status=status.HTTP_400_BAD_REQUEST)
+            return response.error_response()
+
+    def get(self):
+
+        """Provides the data of all the fund transfer"""
+        try:
+            all_fund_transfer = FundTransfer.query.all()
+            output = []
+            for transfer in all_fund_transfer:
+                currentfund = {}
+                currentfund['from_account'] = transfer.from_account
+                currentfund['to_account'] = transfer.to_account
+                output.append(currentfund)
+            logger.info("All fund transfer data returned successfully")
+            response = ResponseGenerator(data=output, message="All fund transfer data returned successfully",
+                                         success=True, status=status.HTTP_200_OK)
+            return response.success_response()
+        except Exception as error:
+            logger.exception(error)
+            response = ResponseGenerator(data={}, message="Invalid request", success=False,
+                                         status=status.HTTP_404_NOT_FOUND)
+            return response.error_response()
+
+
+class FundTransferData(Resource):
+
+    """ for FundTransferData GET(single fund transfer detail),
+        PUT(update fund transfer), DELETE(delete fund transfer)"""
+
+    def get(self, id):
+
+        """Gives the data of single fund transfer  with selected fund transfer id """
+
+        try:
+            fund = FundTransfer.query.filter(FundTransfer.id == id).first()
+            output = fund_transfer_schema.dump(fund)
+            if fund:
+                logger.info('fund transfer returned successfully')
+                response = ResponseGenerator(data=output, message="fund transfer returned successfully",
+                                             success=True, status=status.HTTP_200_OK)
+                return response.success_response()
+            else:
+                logger.exception("fund transfer id not found")
```

Comment:
Try to use custom exception class and raise exception here and handle it below and then generate response from there.

Reply:
have used the custom class exception and handled in all the APIs